app/src/main/res/values/strings.xml

```diff
@@ -156,6 +156,7 @@
     <string name="share_items">Share</string>
 
     <string name="sign_up">Sign Up</string>
+    <string name="signUpToLogin">Existing User?\n Click here to Login</string>
```

Comment:
Strange name. Should be `existing_user_login`

See that every other string is snake case and you have used camel case

Reply:
I will correct my mistake sir soon